hummingbird/ml/_utils.py

```diff
@@ -180,6 +180,47 @@ def from_strings_to_ints(input, max_string_length):
     return np.array(input, dtype="|S" + str(max_string_length)).view(np.int32).reshape(shape)
 
 
+def load(location):
+    """
+    Utility function used to load arbitrary Hummingbird models.
+    """
+    # Add load capabilities
+    from ._container import PyTorchSklearnContainer
+    from ._container import TVMSklearnContainer
+    from ._container import ONNXSklearnContainer
+
+    model = None
+
+    # Try as TVM model.
+    try:
+        model = TVMSklearnContainer.load(location)
```

Comment:
I wondered why you were doing it this way when I learned pickle has[ no header](https://stackoverflow.com/questions/13939913/how-to-test-if-a-file-has-been-created-by-pickle)?  This is clunky but I don't see a good way around.

Reply:
Fixed!